core/layout_api.php

```diff
@@ -271,6 +271,9 @@ function layout_head_css() {
 		html_css_link( 'bootstrap-datetimepicker-' . DATETIME_PICKER_VERSION . '.min.css' );
 	}
 
+	# floating scroll (no cdn available)
+	html_css_link( 'jquery.floatingscroll-' . FLOATING_SCROLL_VERSION . '.css' );
```

Comment:
Why using another library for this? Is this not possible with perfect-scrollbar plugin that I have in use @ https://github.com/mantisbt/mantisbt/pull/1088?

If